pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <groupId>com.mobidevelop.robovm</groupId>
-  <version>2.5.0-8-SNAPSHOT</version>
```

Comment:
Shouldn't need this as maven release will release with 2.5.0-8 and then increment itself to next snapshot.

Reply:
thx, reverted